mobile/src/full/java/org/openhab/habdroid/core/GcmIntentService.java

```diff
@@ -102,12 +103,10 @@ private void sendNotification(String msg, int notificationId) {
         NotificationCompat.Builder mBuilder =
                 new NotificationCompat.Builder(this)
                         .setSmallIcon(R.drawable.ic_openhab_appicon_24dp)
-                        .setContentTitle("openHAB")
-                        .setStyle(new NotificationCompat.BigTextStyle()
-                                .bigText(msg))
```

Comment:
Removing the BigTextStyle has the consequence of no longer being able to expand notifications with longer texts ... I would keep that one.